Spotify UI/lib/helpers/theme.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:flutter/material.dart';
+
+getThemeOf(context) => Theme.of(context);
```

Comment:
This could also be an extension on BuildContext, without requiring a separate file and function